content/en/llm_observability/sdk.md

```diff
@@ -104,6 +104,18 @@ LLMObs.enable(
 : optional - _string_
 <br />The name of the service used for your application. If not provided, this defaults to the value of `DD_SERVICE`.
 
+### Serverless setup
```

Comment:
Should we change this to specify this setup is AWS Lambda specific, or create an AWS lambda section under a server-less section?

Reply:
Ah yeah good catch - I think maybe we can just change it to be AWS Lambda specific. In that case, with respect to your above comment, should we surface the `flush` function a bit more clearly than some smaller FAQ section? If they're not using AWS Lambdas, I would think they want to see the `flush` documentation a little sooner to avoid possible instrumentation pains. Wdyt?

Reply:
Hmm yea good point. i think it should be the bottom-most section in this titled something like "Force Flushing". IMO the description should be generic - "This is a blocking function that force flushes any buffered spans to the Datadog back-end"

Reply:
Ah ok, makes sense. I think that's how it was before, but I thought it seemed too disjoint. But, with this context, I think it makes sense. Will move it and specify this section title 👍

Reply:
yeah, I ended up just putting it back in the same spot.